test/demarcheSimplifiees/verifyPsychologist.spec.js

```diff
@@ -12,7 +11,7 @@ describe('getDiplomaErrors', () => {
     }],
   });
 
-  const getDiplomaErrors = autoVerify.__get__('getDiplomaErrors');
+  const getDiplomaErrors = autoVerifyPsychologists.__get__('getDiplomaErrors');
```

Comment:
You can call the private method this way without autoVerifyPsychologists being rewired?

Reply:
yes, thanks to babel-rewire